game.py

```diff
@@ -529,6 +530,16 @@ def main():
     shoot_counter = 0
 
     while running:
+        ship_data = n.send(p1_server_data_resp)
+        try:
+            modules = ship_data['modules']
```

Comment:
Should put some comments to denote the variable type of the things:
- ship_data = dict[Any] (multiple things)
- `modules` (key) = dict[str, int]
- `weapon` (key)= str  (This is the colour of the element)